ext/odbc/tests/odbc_autocommit_001.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+odbc_autocommit(): Basic test
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+var_dump(odbc_autocommit($conn));
+var_dump(odbc_autocommit($conn, true));
```

Comment:
```suggestion
var_dump(odbc_autocommit($conn, true));
var_dump(odbc_autocommit($conn));
```
Flip those two lines so that we don't depend on the default configuration